images/win/scripts/Installers/Install-Toolset.ps1

```diff
@@ -60,7 +66,7 @@ foreach ($tool in $tools) {
         $asset = $assets | Where-Object version -like $toolVersion `
                          | Sort-Object -Property {[version]$_.version} -Descending `
                          | Select-Object -ExpandProperty files `
-                         | Where-Object { ($_.platform -eq $tool.platform) -and ($_.arch -eq $tool.arch) } `
+                         | Where-Object { ($_.platform -eq $tool.platform) -and ($_.arch -eq $tool.arch) -and ($_.toolset -eq $tool.toolset) } `

```

Comment:
`-and ($_.toolset -eq $tool.toolset)` are you sure about that? As I see, only Boost have toolset field, and this function is used for all toolset tools.

Reply:
yes, it works fine with `-and ($_.toolset -eq $tool.toolset)`.

Reply:
Surely, it will work, but it's just little messy. Maybe we should move platform, arch and additional checks in some separate loop.